waiter/src/waiter/scheduler/kubernetes.clj

```diff
@@ -727,15 +729,20 @@
        :apiVersion replicaset-api-version
        :metadata {:annotations {:waiter/service-id service-id}
                   :labels {:app k8s-name
-                           :waiter-cluster cluster-name}
-                  :name k8s-name}
+                           :waiter-cluster cluster-name
+                           :waiter/user run-as-user}
+                  :name k8s-name
+                  ;; TODO - set namespace via run-in-namespace service parameter
+                  :namespace run-as-user}
        :spec {:replicas min-instances
               :selector {:matchLabels {:app k8s-name
-                                       :waiter-cluster cluster-name}}
+                                       :waiter-cluster cluster-name
+                                       :waiter/user run-as-user}}
```

Comment:
ANy reason we chose `waiter-cluster` over `waiter/cluster` earlier? Does that same restriction still apply for `waiter/user`?

Reply:
Yes, I noticed that too. We should switch to `waiter/cluster`, but I was planning to open a separate PR for that change.

Reply:
Created issue #721 for this.